src/test/java/org/minions/devfund/mauricioz/linkedList/RemoveMaximumValuesTest.java

```diff
@@ -0,0 +1,186 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by Fernflower decompiler)
+//
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RemoveMaximumValuesTest {
+    public RemoveMaximumValuesTest() {
+    }
+
+    /**
+     * setup for tests.
+     */
+    @Before
+    public void setUp() throws Exception {
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testNull() {
+        try {
+            LinkedListUtils.removeMaximumValues((LinkedList)null, 2);
+        } catch (Exception var2) {
+            Assert.fail("removeMaximumValues throws " + var2 + " when input LinkedList is null");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "removeMaximumValues throws " 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)